docs/index.md

```diff
@@ -4,204 +4,119 @@ title: "Introduction to ZIO Logging"
 
-- Type-safe, purely-functional, ZIO-powered
-- Compositional, type-safe log formatting
+@PROJECT_BADGES@
+
+## Introduction
+
+When we are writing our applications using ZIO effects, to log easy way we need a ZIO native solution for logging. ZIO Logging is an environmental effect for adding logging into our ZIO applications.
+
+Key features of ZIO Logging:
+
+- **ZIO Native** — Other than it is a type-safe and purely functional solution, it leverages ZIO's features.
+- **Multi-Platform** - It supports both JVM and JS platforms.
+- **Composable** — Loggers are composable together via contraMap.
+- **Pluggable Backends** — Support multiple backends like ZIO Console, SLF4j, JS Console, JS HTTP endpoint.
+- **Logger Context** — It has a first citizen _Logger Context_ implemented on top of `FiberRef`. The Logger Context maintains information like logger name, filters, correlation id, and so forth across different fibers. It supports _Mapped Diagnostic Context (MDC)_ which manages contextual information across fibers in a concurrent environment.
 - Richly integrated into ZIO 2's built-in logging facilities
 - ZIO Console, SLF4j, and other backends
 
 ## Installation
 
-`ZIO-Logging` is available via maven repo.
-In order to use this library, we need to add the following line in our build.sbt file:
+In order to use this library, we need to add the following line in our `build.sbt` file:
 
 ```scala
-libraryDependencies += "dev.zio" %% "zio-logging" % @VERSION@
+libraryDependencies += "dev.zio" %% "zio-logging" % "@VERSION@"
 ```
 
-### Log Format
-
-A `LogFormat` represents a DSL to describe the format of text log messages.
+There are also some optional dependencies:
 
 ```scala
-import zio.logging.console
-import zio.logging.LogFormat._
-
-val myLogFormat = timestamp.fixed(32) |-| level |-| label("message", quoted(line))
-val myConsoleLogger = console(myLogFormat)
-```
-
-### Logger Context and Annotations
-
-The `logContext` fiber reference is used to store typed, structured log
-annotations, which can be utilized by backends to enrich log messages.
+// SLF4j integration
+libraryDependencies += "dev.zio" %% "zio-logging-slf4j" % "@VERSION@"
 
-Because `logContext` is an ordinary `zio.FiberRef`, it may be get, set,
-and updated like any other fiber reference. However, the idiomatic way to
-interact with `logContext` is by using `zio.logging.LogAnnotation`.
+// Using ZIO Logging for SLF4j loggers, usually third-party non-ZIO libraries
+libraryDependencies += "dev.zio" %% "zio-logging-slf4j-bridge" % "@VERSION@"
 
-For example:
+// Scala.js console integration
+libraryDependencies += "dev.zio" %% "zio-logging-jsconsole" % "@VERSION@"
 
-```scala
-myResponseHandler(request) @@ LogAnnotation.UserId(request.userId)
+// Scala.js HTTP Logger which sends logs to a backend via Ajax POST
+libraryDependencies += "dev.zio" %% "zio-logging-jshttp" % "@VERSION@"
```

Comment:
zio-logging-jsconsole and zio-logging-jshttp, are zio-logging version 1 only,
they was dropped with version 2

Reply:
So what is the plan for version 1 docs? Do you want to maintain separate documentation for version 1?

Reply:
I removed those from the introduction page. Done!